### PR TITLE
add all env variables to README

### PR DESCRIPTION
**Pull request by KPostOffice**
## Related Issues and Dependencies

#262

### CODE REVIEW

**Review by pacospace — APPROVED** [review]
Thanks :clap: :100:

File: README.md [hunk 1]
```diff
@@ -29,6 +29,15 @@ to Kafka for each package release to be solved using Thoth [Solver](https://gith
 
 Consumer is currently able to handle the following Kafka messages focused on:
 
+#### Environment Variables
```
Comment (pacospace) [inline README.md:32]: It should be above `Consumer is currently able to handle the following Kafka messages focused on:` ?

Comment (KPostOffice) [reply README.md:32]: Yeah, my bad

**Review by fridex — COMMENTED** [review]
/lgtm

**Comment by fridex** [comment]
@pacospace I think this can be unheld.

**Pull request merged**